modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -26,6 +26,37 @@ Before installing Anypoint Runtime Fabric in a Kubernetes environment, ensure:
 * If you want to use a local registry with Runtime Fabric, follow the installation instructions in xref:configure-local-registry.adoc[Using a Local Registry with Runtime Fabric]. You must set up the local registry during installation. 
 ====
 
+== RTFCTL Versions Compatibility
+
+The following tables list the `rtfctl` versions compatibility with Runtime Fabric appliance 1.x.x agent versions and Runtime Fabric self-managed 2.x.x agent versions. 
+
+=== Runtime Fabric Appliance 1.x.x
+
+The following `rtfctl` versions are compatible with all supported Runtime Fabric agents on appliance clusters:
+
+[%header%autowidth.spread]
+|===
+| RTF appliance 1.x.x agent version | `rtfctl` version 
+| All versions | 0.6.0-2.release-gravity
+| All versions | 0.5.2-4.release-gravity
+| All versions | 0.3.87 to 0.4.22
```

Comment:
Is this one correct? Because I see that those versions are from [RTFCTL BYOK](https://docs.mulesoft.com/release-notes/runtime-fabric/runtime-fabric-command-line-tool-release-notes) and not from [RTFCTL APPLIANCE](https://docs.mulesoft.com/release-notes/runtime-fabric/runtime-fabric-command-line-tool-release-notes-appliance). Not sure if this applies.